src/components/side-panel/layer-panel/layer-type-selector.js

```diff
@@ -103,7 +103,7 @@ const DropdownListWrapper = styled.div`
 `;
 
 function LayerTypeListItemFactory() {
-  const LayerTypeListItem = ({value, isTile}) => (
+  return ({value, isTile}) => (
```

Comment:
you will probably hahve the `missing displayName` error without the `const LayerTypeListItem=` it's also better for React devTools if we keep the declaration